diagnostic.md

```diff
@@ -150,7 +180,7 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+They are block elements, but the divs with float left will float inline to the top left of the page (not over top of one another), .divOne will be to the top right of the page, and items on the page will not be able to float to the left of .divFour.
```

Comment:
.divOne will be in the top-right corner; .divTwo will be in the top-left corner, with .divThree immediately to its right; .divFour will be underneath .divTwo, and .divFive will sit immediately to the right of .divFour (under .divThree).

In short:
2 3      1
4 5